packages/api/test/e2e/api/promise-tx-eras.spec.ts

```diff
@@ -0,0 +1,135 @@
+// Copyright 2017-2019 @polkadot/api authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import testingPairs from '@polkadot/keyring/testingPairs';
+import WsProvider from '@polkadot/rpc-provider/ws';
+import { EventRecord, Header, Index, SignedBlock } from '@polkadot/types/interfaces';
+import { createType } from '@polkadot/types';
+
+import { SubmittableResult } from '../../../src';
+import ApiPromise from '../../../src/promise';
+import { Signer } from '../../../src/types';
+import { describeE2E, SingleAccountSigner } from '../../util';
+
+// log all events for the transfer, calling done() when finalized
+const logEvents = (done: () => {}): (r: SubmittableResult) => void =>
```

Comment:
Doesn't need to be done now - but this function is shared in a lot of places, should be split.